test/pgtest/copy.pt

```diff
@@ -28,6 +29,8 @@ ReadyForQuery
 ----
 ParseComplete
 BindComplete
+ParameterDescription {"parameters":[]}
+NoData
```

Comment:
This is surprising. Did you generate this from postgres?

Reply:
I did! Matched my expectations—though should I be more surprised? Have to return _something_ for Describe, and this is more consistent with what happens in the basic query protocol.

Reply:
Oh I didn't remember that the response to describe is two things. Yeah this is fine.